test/e2e/upgrade_management_cluster_test.go

```diff
@@ -86,6 +86,7 @@ func preWaitForCluster(clusterProxy framework.ClusterProxy, clusterNamespace str
 	// Split the <fileName> file into <resourceName>.yaml files based on the name of each resource and return list of the filename created
 	// so we can apply limited number on BMH in each namespace.
 	splitYAMLFile := func(fileName string, filePath string) []string {
+		//#nosec G204 -- This is just a test. We are not concerned with command injection
```

Comment:
I didn't see any reasonable way to get rid of this easily right now unfortunately. However, I think we could definitely rework it later so that we do all of it in go. But I would prefer to first get this merged so we avoid adding more issues in the mean time.

Reply:
I did update the comments though to reflect that we do care about this even though it is a test.

Reply:
Thanks for checking. Please create a ticket to follow-up so its not forgotten.